Firestore/Example/Podfile

```diff
@@ -29,12 +28,6 @@ target 'Firestore_Example' do
     pod 'OCMock'
   end
 
-  target 'Firestore_SwiftTests_iOS' do
```

Comment:
Swift targets need to be removed because plain static libraries only work with Objective C.

Reply:
So ... how are we going to test these things?

The Swift build test is a critical component of our verification that our API actually ends up translated correctly. If I had to pick between Objective-C static library support and fully testing our swift interaction I'll pick full Swift support every time. That's where our users are.